### PR TITLE
Revert "Rcal 406 science array quantities"

### PR DESCRIPTION
**Pull request by PaulHuwe**
I somehow merged PR-613 prematurely.

Reverts spacetelescope/romancal#613

### CODE REVIEW

**Comment by PaulHuwe** [comment]
We can do this reversion, or I can just make a new for PR the requested changes to #613

**Review by ddavis-stsci — APPROVED** [review]
LGTM

File: pyproject.toml [hunk 1]
```diff
@@ -12,17 +12,16 @@ classifiers = [
...
+    'asdf >=2.12.1',
     'astropy >=5.0.4',
     'crds >=11.16.16',
     'gwcs >=0.18.1',
     'jsonschema >=3.0.2',
     'numpy >=1.20',
     'pyparsing >=2.4.7',
     'requests >=2.22',
-    #'roman_datamodels >=0.14.0',
+    'roman_datamodels >=0.14.0',
```
Comment (PaulHuwe) [inline pyproject.toml:23]:
```suggestion
    'roman_datamodels =0.14.0',
```

File: pyproject.toml [hunk 2]
```diff
@@ -12,17 +12,16 @@ classifiers = [
...
+    'asdf >=2.12.1',
     'astropy >=5.0.4',
     'crds >=11.16.16',
     'gwcs >=0.18.1',
     'jsonschema >=3.0.2',
     'numpy >=1.20',
     'pyparsing >=2.4.7',
     'requests >=2.22',
-    #'roman_datamodels >=0.14.0',
+    'roman_datamodels =0.14.0',
```
Comment (PaulHuwe) [inline pyproject.toml:23]:
```suggestion
    'roman_datamodels >=0.14.0',
```

**Pull request merged**